packages/blade/src/components/BottomSheet/BottomSheet.stories.tsx

```diff
@@ -404,14 +404,15 @@ export const WithSelect = BottomSheetWithSelectTemplate.bind({});
 WithSelect.storyName = 'WithSelect';
 
 const BottomSheetScrollingContentTemplate: ComponentStory<typeof BottomSheetComponent> = () => {
-  const [isOpen, setIsOpen] = React.useState(true);
+  const [isOpen, setIsOpen] = React.useState(false);
```

Comment:
Do you mean to change this?

Reply:
Reverted

Reply:
Let's keep it false only, set it to true for debugging purposes.